exercises/bowling/bowling.spec.js

```diff
@@ -0,0 +1,467 @@
+import Bowling from './bowling.js';
+
+describe('Bowling', () => {
+  let game;
+
+  const playGame = rolls => {
+    rolls.map(pins => game.roll(pins));
+  };
+
+  beforeEach(() => {
+    game = new Bowling();
+  });
+
+  test('Calculate score for a game with all zeros', () => {
+    const rolls = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
+    playGame(rolls);
+    expect(game.score()).toBe(0);
+  });
+
+  xtest('Calculate score for a game with no strikes or spares', () => {
+    const rolls = [3, 6, 3, 6, 3, 6, 3, 6, 3, 6, 3, 6, 3, 6, 3, 6, 3, 6, 3, 6];
+    playGame(rolls);
+    expect(game.score()).toBe(90);
+  });
+
+  xtest('A game with a spare followed by zeros is worth ten points', () => {
+    const rolls = [6, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
+    playGame(rolls);
+    expect(game.score()).toBe(10);
+  });
+
+  xtest('Points scored after spare are counted twice', () => {
+    const rolls = [6, 4, 3, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
+    playGame(rolls);
+    expect(game.score()).toBe(16);
+  });
+
+  xtest('Consicutive spares get one round bonus each', () => {
+    const rolls = [5, 5, 3, 7, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
+    playGame(rolls);
+    expect(game.score()).toBe(31);
+  });
+
+  xtest(
+    'A spare in the last frame gets a bonus roll that is counted ones',
+    () => {
+      const rolls = [
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        0,
+        7,
+        3,
+        7,
```

Comment:
I'm not sure how we have our ESlint rules set up for this project-- do you think your plugin is doing this because of our rules, or in spite of them?  Maybe we need to address this in our config for the repo.